packages/@react-aria/menu/stories/useMenu.stories.tsx

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
This is just a useful story for testing, i can move it into its own PR if we want